command/command.go

```diff
@@ -478,6 +479,7 @@ func runCheckersLoop(app *App, termCheckerCh <-chan struct{}, quit <-chan struct
 			select {
 			case report := <-checkReportCh:
 				reports = append(reports, report)
+			case <-reportImmediateCh: // drain all
```

Comment:
[q]
Since we already received from `reportImmediateCh` in L472, `reportImmediateCh` should have been inserted (at least) twice to receive  here?

Reply:
reportImmediateCh has a buffer to avoid blocking checks.
Since it is unnecessary except the first one, drain all.